packages/hawtio/src/ui/icons/TooltipHelpIcon.tsx

```diff
@@ -3,7 +3,7 @@ import { HelpIcon } from '@patternfly/react-icons'
 
 export const TooltipHelpIcon: React.FunctionComponent<{ tooltip: string }> = ({ tooltip }) => (
   <Icon size='md'>
-    <Tooltip content={tooltip} removeFindDomNode>
+    <Tooltip content={tooltip}>
```

Comment:
is it intentional? no more needed?

Reply:
yes, as I mentioned above, it's not longer needed